src/ol/controls/RoutingControl.js

```diff
@@ -61,24 +66,35 @@ class RoutingControl extends Control {
       },
     });
 
+    /** @ignore */
     this.mot = options.mot || 'bus';
 
     this.abortController = new AbortController();
 
+    /** @ignore */
     this.apiKey = options.apiKey;
 
+    /** @ignore */
+    this.stopsApiKey = options.stopsApiKey || this.apiKey;
+
+    /** @ignore */
+    this.stopsApiUrl =
+      options.stopsApiUrl || 'https://api.geops.io/stops/v1/lookup/';
+
```

Comment:
Should the lookup url use the this.url property ?

Reply:
yes. Maybe the  name for the variable is a bit confusing, since we don's actually use the StopsAPI class